extensions/hints/init.lua

```diff
@@ -114,7 +149,14 @@ function hints.displayHintsForDict(dict, prefixstring, showTitles)
         else
           local suffixString = ""
           if showTitles then
-            suffixString = ": "..win:title()
+            win_title = win:title()
```

Comment:
I flipped math.min to math.max, I think that makes more sense to maximize the end_idx value. Everything I took your suggestions as is, thanks!